pkg/providers/instancetype/suite_test.go

```diff
@@ -155,27 +155,81 @@ var _ = Describe("InstanceTypeProvider", func() {
 				},
 			},
 		})
-		nodeClass.Status.SecurityGroups = []v1beta1.SecurityGroup{
-			{
-				ID: "sg-test1",
+
+		nodeClass.Status = v1beta1.EC2NodeClassStatus{
+			InstanceProfile: "test-profile",
+			AMIs: []v1beta1.AMI{
```

Comment:
Do we need the AMIs section here?

Reply:
This is to help with merge conflict, but I can remove it